.github/workflows/tests.yml

```diff
@@ -45,7 +45,7 @@ jobs:
           sccache --show-stats
           echo "########## START UNIT TESTS ##########"
           cargo test
-  selected-functionnal-tests:
+  selected-functional-tests:
```

Comment:
Do not update this one, it requires a change on our end as well, better to do it in another PR.

Reply:
Sure